README.md

```diff
@@ -80,6 +80,42 @@ These commands should work:
     sudo zypper in aspell-devel aspell-en ruby-devel
     sudo gem install raspell
 
+## server
+Runs a simple web server which provides a dynamically generated tarball with
+the source code. The web server is designed to serve the source tarballs for
+the `yupdate` script which can easily update the YaST code from your local
+Git checkout.
+<!-- TODO: add a link to the yupdate script later -->
```

Comment:
I would like to see a real link here, even if it may change in the future.

Also, a matching `yupdate foo bar:8000` incantation would be helpful.